cmd/controller/disk.go

```diff
@@ -78,6 +81,11 @@ type ProbeIdentifier struct {
 	SeachestIdentifier string // SeachestIdentifier (devPath) is used to identify disk by seachest.
 }
 
+type PartitionInfo struct {
+	PartitionType string // Partition type like 83, 8e etc.
+	FileSystem    string // Filesystem of partition. can be none, xfs etc.
```

Comment:
Can we have enum to name a particular filesystem

Reply:
I think you meant using `const`. But that means I will have to store name of every filesystem right? Is it a good approach?